expo-zustand-styled-components/src/screens/Organization/Organization.styles.ts

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/native';
+import { breakpoints } from '../../utils/breakpoints';
+
+export const SafeAreaViewStyled = styled.SafeAreaView`
+  height: 100%;
+  background-color: #fff;
+`;
+
+export const ContainerStyled = styled.ScrollView<{ screenWidth: number }>`
+  flex: 1;
+  width: 100%;
+  height: 100%;
+  position: relative;
+  flex-direction: column;
+  padding-horizontal: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? '15%' : 0)};
+`;
+
+export const OrgInfoContainer = styled.View`
+  gap: 6px;
+  display: flex;
+  padding: 16px;
+  align-items: center;
+  flex-direction: row;
+`;
+
+export const TabNavContainer = styled.View`
+  padding-top: 8px;
+  padding-horizontal: 16px;
+  border-bottom-width: 1px;
+  border-bottom-color: #e1e4e8;
```

Comment:
I believe this color exists in the colors object, if it doesn't you can update it so we can reuse it where appropriate.

What do you think?